modules/core/src/main/java/org/apache/synapse/config/xml/SynapsePathSerializer.java

```diff
@@ -107,6 +107,25 @@ public static OMElement serializeTextPath(SynapsePath path, String expression,
 		return elem;
 	}
 
+    public static OMElement serializePath(SynapsePath path, OMElement elem, String attribName, char brace) {
+
+        OMNamespace nullNS = elem.getOMFactory()
+                .createOMNamespace(XMLConfigConstants.NULL_NAMESPACE, "");
+
+        if (path != null) {
+
```

Comment:
Remove unnecessary empty lines. Format the code.

Reply:
I thought of keeping this the same way because all the other methods in class are having empty lines like this. But even then if you want, i will remove them.